api/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10 AS builder
+
+ARG TARGETARCH
+USER root
+RUN microdnf install -y tar gzip unzip
+
+# install platform specific go version
+RUN curl -O -J  https://dl.google.com/go/go1.22.0.linux-${TARGETARCH}.tar.gz
+RUN tar -C /usr/local -xzf go1.22.0.linux-${TARGETARCH}.tar.gz
+RUN ln -s /usr/local/go/bin/go /usr/local/bin/go
+
+
+# Install protoc
+RUN curl -L -o protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v27.1/protoc-27.1-linux-x86_64.zip \
```

Comment:
I also wonder how we are going to keep on top of newer versions. I don't have the answer, but it would be great if we could somehow have dependabot do something for us.

Reply:
I can look into it, might be more difficult with keeping everything up to date with the java version.

Reply:
Don't let this comment block a merge.